BTrees/tests/testBTrees.py

```diff
@@ -361,7 +358,7 @@ def test_LP294788(self):
         for id in ids:
             del t[id]
         #release all refs
-        ids = obj = id = None
+        ids = id = None
```

Comment:
I'm assuming `obj` was otherwise unused in this method?

Reply:
That's correct.